Tests/test_file_avif.py

```diff
@@ -233,7 +233,7 @@ def test_background_from_gif(self, tmp_path: Path) -> None:
         with Image.open(out_gif) as reread:
             reread_value = reread.convert("RGB").getpixel((1, 1))
         difference = sum([abs(original_value[i] - reread_value[i]) for i in range(3)])
-        assert difference <= 3
```

Comment:
`original_value` is `(254, 255, 252)`
Within libavif 1.2.1, `reread_value` is `(253, 254, 251)`. With libavif 1.3.0, it is `(252, 254, 249)`. This is still close to the original value.